7.10.10/Dockerfile

```diff
@@ -68,6 +68,15 @@ RUN composer install --no-plugins --no-scripts
 COPY php.custom.ini /usr/local/etc/php/conf.d/
 COPY ./bootstrap /bootstrap
 
+# unsafe keys
+COPY p*.key  Api/V8/OAuth2/
```

Comment:
Please don't check private/public keys into version control. This key pair is completely internal to SuiteCRM. They're used to create JWT tokens for authenticating calls to SuiteCRM's API.

They can be generated when the container is started for the first time. I think that the `ENTRYPOINT` directive would be a better fit for this use case.

What do you think?

Reply:
About keys, I understand that requirement 1 is: user can start everything rapidly with minimal number of commands.
I imagine requirement 2: The OAuth2 keys should be retained if user upgrades the image to new version. (Existing tokens are still valid.)

I'll do it Docker-secret-ready. If user sets docker secrets, his OAuth2 clients which hold tokens will not be impacted by on-the-fly image upgrade.

Can I use `CMD` instead of `ENTRYPOINT`? We're already overriding parent's `CMD` anyway.

Please confirm requirement 2 and I'll proceed.

Reply:
> The OAuth2 keys should be retained if user upgrades the image to new version. (Existing tokens are still valid.)

That's a nice catch Jakub. In this particular case however, it's not a problem. SuiteCRM reports HTTP 401 in that case when the token is expired and also when the token is invalid due to changed keys.

The client code calling this API will take care or requesting a new token in both cases.

> Can I use CMD instead of ENTRYPOINT? We're already overriding parent's CMD anyway.

I'd leave this decision to your good judgement. You're the Docker expert after all. :)